src/Stryker.Core/Stryker.Core.UnitTest/Reporters/HtmlReporterTests.cs

```diff
@@ -124,6 +147,30 @@ public void ShouldOpenHtmlReportIfOptionIsProvided()
             mockProcess.Verify(m => m.Open(reportUri));
         }
 
+        [Fact]
+        public void ShouldOpenHtmlReportIfOptionIsProvidedAndSpacesInPath()
+        {
+            var mockProcess = new Mock<IWebbrowserOpener>();
+            var mockFileSystem = new MockFileSystem();
+            var options = new StrykerOptions
+            {
+                ReportTypeToOpen = ReportType.Html,
+                OutputPath = " folder \\ next level",
+                ReportFileName = "mutation-report"
+            };
+
+            var reporter = new HtmlReporter(options, mockFileSystem, processWrapper: mockProcess.Object);
+            var mutationTree = ReportTestHelper.CreateProjectWith();
+
+            reporter.OnAllMutantsTested(mutationTree);
+
+            var reportUri = Path.Combine(options.ReportPath, $"{options.ReportFileName}.html");
+            reportUri = "file://" + reportUri.Replace("\\", "/");
+
+            // Check if browser open action is invoked
+            mockProcess.Verify(m => m.Open(reportUri));
```

Comment:
```suggestion
            mockProcess.Verify(m => m.Open(reportUri));
            mockProcess.VerifyNoOtherCalls();
```

Should add VerifyNoOtherCalls since mock is not setup as Strict.